spring-cloud-aws-autoconfigure/src/main/java/io/awspring/cloud/autoconfigure/sqs/SqsAutoConfiguration.java

```diff
@@ -101,15 +104,22 @@ public SqsMessageListenerContainerFactory<Object> defaultSqsListenerContainerFac
 		errorHandler.ifAvailable(factory::setErrorHandler);
 		interceptors.forEach(factory::addMessageInterceptor);
 		asyncInterceptors.forEach(factory::addMessageInterceptor);
-		objectMapperProvider.ifAvailable(objectMapper -> setObjectMapper(factory, objectMapper));
+		objectMapperProvider.ifAvailable(objectMapper -> setObjectMapper(factory, objectMapper, messagingMessageConverter));
```

Comment:
I might be missing something, but in this case wouldn't the converter only be set if we also have an object mapper?

Reply:
Thank you for the review. As you mentioned, using ObjectProvider<ObjectMapper> is to account for the possibility that ObjectMapper might not be available. I have modified the code so that messagingMessageConverter is set even if ObjectMapper is not present. I have updated the PR with these changes. Please review it.